RS-Project-6/NQueensProblem/NQueensTest.java

```diff
@@ -0,0 +1,30 @@
+import static org.junit.Assert.*;

```

Comment:
Make sure to cover every scenario in the Test class. Positive as well as Negative